portal/src/texts/designer/gettingStarted.md

```diff
@@ -1,19 +1,25 @@
 ## Kom i gang
 
-Er du utvikler eller designer og ivrig etter å komme i gang, finner du de ressursene du trenger her. Vi ønsker at det skal være så enkelt så mulig for dere å ta i bruk Jøkul.
+Er du utvikler eller designer og ivrig etter å komme i gang? Her finner du de ressursene du trenger! Vi ønsker at det skal være så enkelt så mulig for alle å ta i bruk Jøkul.
 
 ### Om Jøkul
 
-Jøkul er designsystemtet for Fremtind. Det utvikles stadig og målet med systemet er at det skal gjøre det lettere for alle i Fremtid å utvikle og vedlikeholde løsningene sine.
+Jøkul er designsystemet til Fremtind. Det er under stadig utvikling,og målet med systemet er at det skal gjøre det lettere for alle fagteam i Fremtind å utvikle og vedlikeholde løsningene sine.
```

Comment:
Missing space after `stadig utvikling` in the second sentence.

Reply:
Always something:-) I also need a proofreader :-). Will fix.

Reply:
Fixed, but the space was there, lfberge?

Reply:
I mean @piofinn